README.md

```diff
@@ -1,2 +1,154 @@
-# Admin-CLI
-Client
+Client 1.0.0
+<h1 align="center"> Admin-CLI </h1>
+
+<p align="center">This program is about checking students work in ta's computer and send students score to server by CLI</p>
+
+## Documentation
+* [Installation](#installation)
+* [Getting start](#getting-start)
+  - [Available Commands](#available-commands)
+  - [Login](#login)
+  - [Init work directory](#init-work-directory)
+  - [Start checking work](#start-checking-work)
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install
+
+### library we use that are not in python standard library
+$ pip install requests
+
+$ pip install -U click
+
+Let's start the installation process first you need to install library that we mention 
+then we will clone this github into your local computer
+
+ssh 
+> $ git clone git@github.com:ta-assistant/TA-CLI.git
+
+https 
+> $ git clone https://github.com/ta-assistant/TA-CLI.git
+
+and you need to goto TA-CLI directory to install CLI.
+
+
+Goto TA-CLI directory 
+```bash
+$ cd TA-CLI
+```
+install CLI ( for more infomation -> [CLI](https://github.com/ta-assistant/TA-CLI/tree/master/ta_cli#readme) )
+
+```bash
+$ pip install --editable .
+```
+
+Now you have completed the installation process.
+
+## Getting start
+#### Available Commands
+  1. init    [Init TA's work directory Args: workID (str): Work's ID]
+  2. login   [Login]
+  3. start   [Start working on TA directory]
+  4. submit  [Submit]
+
+#### Login
+first let login we login with apikey that you need to ask server manager to get it then assume that your apikey is `1234567890`
```

Comment:
Suggestion:
We will need an APIkey, which you need to get from the server manager. Let's assume that our password is '123456789'.